tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -254,6 +258,93 @@ def test_get_fourier_models_data(
             assert fourier_modes_data.max().max() <= 1
             assert fourier_modes_data.min().min() >= -1
 
+    def test_channel_contributions_forward_pass_recovers_contribution(
+        self, mmm_fitted: DelayedSaturatedMMM
+    ) -> None:
+        channel_data = mmm_fitted.X[mmm_fitted.channel_columns].to_numpy()
+        channel_contributions_forward_pass = (
+            mmm_fitted.channel_contributions_forward_pass(channel_data=channel_data)
+        )
+        channel_contributions_forward_pass_mean = (
+            channel_contributions_forward_pass.mean(axis=(0, 1))
+        )
+        channel_contributions_mean = mmm_fitted.fit_result[
+            "channel_contributions"
+        ].mean(dim=["draw", "chain"])
+        assert (
+            channel_contributions_forward_pass_mean.shape
+            == channel_contributions_mean.shape
+        )
+        np.testing.assert_array_almost_equal(
+            x=channel_contributions_forward_pass_mean / channel_contributions_mean,
+            y=mmm_fitted.y.max(),
+        )
```

Comment:
Interesting equivalence. Is this generally true or just because of the specific toy MMM used.

Out of curiosity (although it could make a good comment in the test), why is it true?

Reply:
The logic of the test is: the forward pass should be in the original scale. The trace fits the model in with scaled data, so when scaling back, they should match. We are using a `MaxAbsScaler` (see https://github.com/pymc-labs/pymc-marketing/blob/main/pymc_marketing/mmm/delayed_saturated_mmm.py#L379), so it should work for all models (not this specific toy dataset) :) 

Do you think it is worth adding this comment to the test?

Reply:
> Do you think it is worth adding this comment to the test?

Definitely, documenting the logic of tests is very valuable for future devs. Not only for debugging but also during refactoring, to understand if a test is still relevant and not.

Reply:
I added them as it is not trivial from the test. (https://github.com/pymc-labs/pymc-marketing/pull/313/commits/c48adc055c0f8471fecdc6babc3498859cf34d16)